site/docs/hosting/vps.md

```diff
@@ -2,24 +2,116 @@
+If you cannot or do not want to rent a server, you can replace it with a virtual machine.
+To do this, use an application such as [VirtualBox](https://virtualbox.org/).
+Create a virtual machine with the desired Linux distribution and use it as a real server.
+:::
+
+Go to the [VPS tariff catalog](https://hostinger.com/vps-hosting/).
+We will use the "VPS 1" plan.
+Below you can see a comparison table that describes the offered server resources and limits.
+The resources of VPS 1 are enough for bots with a small audience, and even more so for our test bot.
+
+Go back to the "VPS 1" card and click the "Add" button.
+You will be automatically redirected to the checkout page, where you will also immediately register with Hostinger.
+
+:::warning Change the rental term!
+The typical lease term is 2 years (a marketing ploy), and it costs a lot of money.
+You probably don't need it, so you can start by renting a server for a month, which is much cheaper.
+
+In any case, Hostinger provides a 30-day money-back guarantee.
+:::
+
+After payment you will be able to set up your server:
+
+1. **Location.**
+   We recommend that you choose the location closest to Amsterdam.
+   It is in Amsterdam that the Telegram server is located, which is responsible for ensuring the operation of bots.
+   If you use your own Bot API server, choose the location closest to it.
+2. **Server type.**
+   Choose the option "Clean OS".
+3. **Operating system.**
+   We will use Ubuntu 22.04.
+   If you choose a different system, some steps may be different, so be careful.
+4. **Server name.**
+5. **Root password.**
+   Make a strong password and keep it in a safe place!
+6. **SSH key**.
+   Skip this step.
+   We will set up SSH keys [later](#ssh-keys).
+
+After the server is created, you can connect to it using SSH:
+
+> SSH (_Secure Shell_) is a network protocol for remote control of a computer (server) and file transfer.
+
+```sh:no-line-numbers
+ssh root@<ip-address>
+```
+
+replacing `<ip-address>` with the IP address of your server, which you can find on the server management page.
+
+:::tip Separate User for Each Application
+In this guide, all actions with the server will be performed as the root user.
+This is done on purpose to simplify the material.
+However, in reality, the root user should only be responsible for general services (web server, database, etc.), and applications should be run by individual non-root users.
+This approach ensures the security of confidential data and prevents hacking of the entire system.
+At the same time, it imposes some inconveniences.
+Describing all these points unnecessarily increases the complexity of the article, which we try to avoid.
+:::
+
+## Starting the Bot
+
+We now have a server at our disposal where we can run the bot to keep it running around the clock.
 
-## systemd
+> To simplify the beginning of the article, we skipped the step of delivering the code to the server, but it is described [below](#ci-cd).
```

Comment:
Should we leave a quick note how to get the files to the server using `scp`?
```suggestion
> To simplify the beginning of the article, we skipped the step of delivering the code to the server automatically each time after pushing your code, but it is described [below](#ci-cd).
```

Reply:
```suggestion
> To simplify the beginning of the article, we skipped the step of delivering the code to the server automatically each time after pushing your code, but it is described [below](#ci-cd).
>
> For now, you can copy local files to a remote server using the following command:
>
> ```sh:no-line-numbers
> scp <path-to-local-files> root@<ip-address>:<path-to-remote-directory>
> ```
>
> replacing `<path-to-local-files>` with the path to local files, `<ip-address>` with the IP address of your server, and `<path-to-remote-directory>` with the path to the directory where the bot's source code should be stored on the server.
```

@KnorpelSenf like this?

Reply:
Yep!